arch/risc-v/Kconfig

```diff
@@ -82,13 +106,19 @@ config ARCH_CHIP_MPFS
 
 config ARCH_CHIP_RV32M1
 	bool "NXP RV32M1"
+	select ARCH_RV32
+	select ARCH_RV_ISA_M
+	select ARCH_RV_ISA_C
 	select ARCH_RV32IM
 	---help---
 		NXP RV32M1 processor (RISC-V Core with PULP extensions).
 
 config ARCH_CHIP_QEMU_RV32
 	bool "QEMU RV32"
-	select ARCH_RV32IM
+	select ARCH_RV32
+	select ARCH_RV_ISA_M
+	select ARCH_RV_ISA_C
+	select ARCH_RV32IM # TODO: Remove this old style options
```

Comment:
why not remove now

Reply:
Done, ARCH_RV32IM and ARCH_RV64GC had been removed.